methods/get_seqrun_meta.nf

```diff
@@ -1,9 +1,21 @@
 // Function for sequencing_run & clarity_sample_id
 def get_seqrun_meta(LinkedHashMap row) {
```

Comment:
Not that it was introduced here, but I think it would be great with a clearer description of what this method does, while we're at it. Right now have to guess a bit.